assets/app/views/other-resources.html

```diff
@@ -0,0 +1,87 @@
+<project-header class="top-header"></project-header>
+  <project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div id="scrollable-content" class="middle-container has-scroll">
+      <div class="middle-header header-light">
+        <div class="container-fluid">
+          <div class="page-header page-header-bleed-right page-header-bleed-left">
+            <h1>Other Resources</h1>
+          </div>
+          <alerts alerts="alerts"></alerts>
+          <div ng-if="!renderOptions.showGetStarted" class="data-toolbar">
```

Comment:
nope it must have been copied over to the services page too because thats where i copied from